metabase/metabase
# Revert "Fix card filter connected to a field not causing an update"

Reverts #36206 — it introduced a regression making dashboards rerun all the queries when a new dashboard gets added

### To Verify

1. Open any dashboard with a few cards on it
2. Edit > Add question > Pick a question
3. Ensure the dashboard queries ran only once and that step 2 didn't trigger any reloads

### Demo

**Current**

https://github.com/metabase/metabase/assets/17258145/b55df3e0-52fe-4aa2-aa7f-f2c2fdd2db57

**Reverted**

https://github.com/metabase/metabase/assets/17258145/c12658ca-f923-40fe-bafd-5b01f0f3b586
